README.md

```diff
@@ -7,9 +7,9 @@
 To add the Dremio Community HBase Connector to the list of Dremio sources:
 
 1. From the project root, run `mvn clean install`. 
-   The HBase Connector `dremio-hbase-plugin-4.2.2-202004211133290458-b550b6fa.jar` appears in the `target` directory.
+   The HBase Connector `dremio-hbase-plugin-4.7.3-202008270723550726-918276ee.jar` appears in the `target` directory.
```

Comment:
I suggest that we replace with a token such as <version string> so we don't have to update each time we upgrade.

Reply:
I read through a couple Github markdown guides, I didn't find a way to make static text variables. Can you elaborate what you meant be tokenizing the version number?

Reply:
I just meant put placeholder text and let the user figure out the number.

Reply:
Updated.